src/reducers/books.js

```diff
@@ -0,0 +1,38 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/index';
+
+const initialState = {
+  books: [
+    { id: Math.random(), title: 'I am Malala', category: 'Action' },
+    { id: Math.random(), title: 'The Survivors', category: 'Horror' },
+    { id: Math.random(), title: 'Ghost Rider', category: 'History' },
+    { id: Math.random(), title: 'Ghost Rider', category: 'Biography' },
+  ],
+};
```

Comment:
Good work.

Yes, the reducer is first initialized here but in React you can always access the state anywhere in your code, keep in mind reducer is for handling actions.

Kindly improve your code by creating the book example in `src.index.js`

From Pathwright;
![image](https://user-images.githubusercontent.com/31897434/121162540-eb3eeb80-c845-11eb-9520-a44e2e225633.png)

Let's keep to the instructions given on pathwright to reduce time spent on creating CRR, and also we both know it is mandatory for all students.